### ISSUE TITLE
Reorganize help screen

### ISSUE THREAD

**Issue by FastThenLeft** [nicolargo/glances#2037]
**Is your feature request related to a problem? Please describe.**
It's difficult to find commands on the help screen.

**Describe the solution you'd like**
I reorganized the help screen in a document I use. I've grouped the help into four sections:
1. Sort processes:
2. Show/Hide Section: these are commands that actually remove or show a section of data on the screen.
3. Toggle Data Type: these are commands that keep the data on the screen but show it in a different form.
4. Miscellaneous: are commands that didn't fit into the above groupings.

Inside each grouping, I sorted by letter (regardless of case) and then numbers followed by symbols. I also removed a lot of extraneous words to make it quicker to find the command I want. I also capitalized HELP and QUIT so they would stand out more.

This version fits into the same number of lines as the current help screen! Here's the reorganization:

![image](https://user-images.githubusercontent.com/8302277/164456474-bcd7f9ef-e013-455e-89e4-aac7889eae40.png)

**Describe alternatives you've considered**
I had another version, but it was more lines and spaced out more. This version replicates the current version but is easier to find the commands you desire.

What I need to know is if you like it and want any changes. I'm going to try and code it up myself, but I don't know python. I've already made the changes I believe need to be made in glances_help.py. I'm now trying to figure out how to compile/run the whole thing to see if it works.

**Comment by FastThenLeft** [comment]
If you need it more compact, there is one other format that's not too bad:

![image](https://user-images.githubusercontent.com/8302277/164162222-279416dd-d86a-47a8-b6a6-58d7e6be3346.png)

**Comment by RazCrimson** [comment]
@FastThenLeft Nice addition indeed. Would love to see it merged.

To run the glances:

1. Install the dependencies. Would recommend creating a virtualenv and activating it before this to keep things clean.
 ```sh
 pip install -r requirements.txt
 ```
2. Run the glances module at the project root
```sh
python -m glances
```

Feel free to ping me in case you need any help.

**Comment by RazCrimson** [comment]
@FastThenLeft 
Any updates on this? 🤔

**Comment by FastThenLeft** [comment]
Not yet. I'm learning a lot and still plan to do it. I'm playing with venv
right now.

**Comment by FastThenLeft** [comment]
Thanks for waiting on this. Here's a screenshot of how it looks:

```
Glances 3.2.7_beta01 with psutil 5.9.1
Configuration file: None

SORT PROCESSES:                          3  Quick Look                        
  a  Automatically                       4  CPU, MEM, and SWAP                
  c  CPU%                                5  ALL                               
  i  I/O rate                          TOGGLE DATA TYPE:
  m  MEM%                                b  Network I/O: bits/bytes           
  p  Process name                        B  Disk I/O: count/rate
  t  TIME                                F  Filesystem space: used/free       
  u  USER                                S  Quick Look: bar/sparkline         
SHOW/HIDE SECTION:                       T  Network I/O: separate/combined    
  A  Application monitoring              U  Network I/O: live/cumulative      
  d  Disk I/O                            0  Load: Linux/percentage            
  D  Docker                              1  CPU: individual/combined          
  e  Top extended stats                  6  GPU: individual/combined          
  f  Filesystem                          /  Process names: short/full         
  G  GPU                               MISCELLANEOUS:                         
  I  IP                                  E  Erase process filter              
  K  TCP                                 g  Generate history graphs           
  l  Alert logs                          h  HELP                              
  n  Network                             j  Accumulate processes by program   
  N  Time                                k  Kill process                      
  Q  IRQ                                 M  Reset processes summary min/max   
  R  RAID                                q  QUIT (or Esc or Ctrl-C)           
  s  Sensors                             r  Reset history                     
  W  Wifi                                w  Delete warning alerts             
  z  Processes                           x  Delete warning & critical alerts  
  2  Left sidebar                        ENTER: Edit process filter pattern
```
Are there any changes that need to be made?

**Comment by nicolargo** [comment]
Just merge the PR into develop. Thanks @FastThenLeft !

So it is ok for the terminal UI, but it breaks the WebUI:

Before the PR:

![image](https://user-images.githubusercontent.com/776747/177269587-bce07939-f775-4620-85b4-cf08122ecced.png)

After the PR:

![image](https://user-images.githubusercontent.com/776747/177269373-6a57fbae-560d-4882-a434-a37a9fdc8398.png)

**Comment by nicolargo** [comment]
WebUI updated in the last commit.

Thanks @FastThenLeft for the issue record and PR :)

**Comment by FastThenLeft** [comment]
Nicolas-

I'm hoping this just goes to you but it's ok if it doesn't.

I started my career as a programmer, but ended in management. So I missed
all the cool things like Python and git. Now that I'm retired I wanted to
learn them and get back into programming, but I really needed a project to
motivate me. You assigning me that task was the motivation I needed to get
off my butt! Now I am fluent in git (I'm also using it for an Arduino
library I'm building) and going through a course in Python. I just wanted
you to know how that one little act of assigning me that task changed my
life!

BTW, that was my very first PR. I hope there are many more in my life.

-David
